maat-court-data-api/src/main/resources/logback.xml

```diff
@@ -1,7 +1,7 @@
 <configuration>
     <appender name="Console" class="ch.qos.logback.core.ConsoleAppender">
         <encoder>
-            <pattern>%d{HH:mm:ss.SSS} [%thread] traceId: %X{traceId:-} spanId: %X{spanId:-} maat-id:%X{maatId} - laaTransactionId:%X{laaTransactionId} %-5level %logger{36} - %msg%n</pattern>
+            <pattern>%d{HH:mm:ss.SSS} [%thread] %-5level %logger{36} - %msg[%X]%n</pattern>

```

Comment:
traceId: %X{traceId:-} spanId: %X{spanId:-} has been configured as a part of Spring Boot distributed tracing configuration. Could you please add that back.